main/lock.py

```diff
@@ -19,6 +19,7 @@ class RedisLockKey(models.TextChoices):
 
     OPERATION_LEARNING_SUMMARY = _BASE + "-operation-learning-summary-{0}"
     OPERATION_LEARNING_SUMMARY_EXPORT = _BASE + "-operation-learning-summary-export-{0}"
+    DREF_TRANSLATION = _BASE + "-dref-translation-{0}"
```

Comment:
we also need to add model table name in the key...

Reply:
Change DREF_TRANSLATION to MODEL_TRANSLATION
"-{db_table_name}-translation-{0}"